model/Course.js

```diff
@@ -30,7 +30,10 @@ const courseSchema = new Schema({
   isTransfer: { type: Boolean, default: false },
   ratings: Array,
   distribution_ids: [
-    { type: Schema.Types.ObjectId, ref: 'Distribution', required: true },
+    { type: Schema.Types.ObjectId, ref: 'Distribution', default: [] },
+  ],
+  fineReq_ids: [
+    { type: Schema.Types.ObjectId, ref: 'FineRequirement', default: [] },
```

Comment:
take note of which dist / finereqs a. course is part of 

for future projects like looking up what distributions a course satisfies or vice versa--all the courses that satisfy a distribution